tasks/agent.py

```diff
@@ -108,12 +106,8 @@ def build(ctx, rebuild=False, race=False, build_include=None, build_exclude=None
             if ex not in build_exclude:
                 build_exclude.append(ex)
 
-    # remove all tags that are only available on debian distributions
-    distname = distro.id().lower()
-    if distname not in REDHAT_DEBIAN_SUSE_DIST:
-        for ex in REDHAT_DEBIAN_SUSE_ONLY_TAGS:
-            if ex not in build_exclude:
-                build_exclude.append(ex)
+    # remove all tags that are not available for current distro
+    build_exclude.extend(get_distro_exclude_tags())
```

Comment:
`get_distro_exclude_tags()` might potentially add duplicate elements to `build_exclude`, but should be fine since `get_build_tags(build_include, build_exclude)` will handles duplicates.